tests/acquia_lift.test

```diff
@@ -3770,4 +3775,141 @@ class AcquiaLiftWebTestCampaignWizard extends AcquiaLiftWebTestBase {
     $this->drupalPost(NULL, NULL, 'Edit');
     $this->assertUrl('node');
   }
+
+  /**
+   * Verify that audience-style targeting is only available for Acquia Lift
+   * Target agents.
+   */
+  function testAudienceAvailability() {
+    $test_agent = $this->createTestAgent();
+    $target_agent = $this->createTestAgent(array('agent_type' => 'acquia_lift_target'));
```

Comment:
There's now a createTargetingAgent() method that you could use instead. The createTestAgent method is very much geared towards acquia_lift agents.